config/web.php

```diff
@@ -14,4 +16,12 @@
             'params' => $params['yiisoft/yii-swagger']['ui-params'],
         ],
     ],
+    SwaggerJson::class => static function (Injector $injector) use ($params) {
```

Comment:
`Injector` is transitive dependency. Better use dependencies of `SwaggetJson` instead `Injector`.

For this case, solution of issue https://github.com/yiisoft/config/issues/113 can help in future.

Reply:
Done.